Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
```

Comment:
This guide is about etcd in general, not CoreOS. I don't think the patronizing disclaimer about `kill` is necessary; the rest of the guide is written assuming no init system, so what's the problem?